actions/docs-verifier/src/MarkdownLinksVerifier/MarkdownFilesAnalyzer.cs

```diff
@@ -37,13 +38,17 @@ public static class MarkdownFilesAnalyzer
                 }
 
                 MarkdownDocument document = Markdown.Parse(await File.ReadAllTextAsync(file), s_pipeline);
-                foreach (LinkInline link in document.Descendants<LinkInline>())
+                foreach (LinkInline link in
+                    document.Descendants<LinkInline>().Where(link => link.Url is not null))
```

Comment:
Do we have cases where `link.Url` is null?

Reply:
No, this is not related to an issue but rather a compiler warning (on a clean build).